R/stat-smooth.R

```diff
@@ -187,3 +187,6 @@ StatSmooth <- ggproto("StatSmooth", Stat,
 
   required_aes = c("x", "y")
 )
+
+# This function exists to silence an undeclared import warning
+gam_method <- function() mgcv::gam
```

Comment:
For some reason, windows machines didn't pick up on `mgcv::gam()` if it was used in a ggproto method, but it is fine if it is in a regular function body.